<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>606</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="app.qrc">
    <normaloff>:/media/media/DansGame.png</normaloff>:/media/media/DansGame.png</iconset>
  </property>
  <widget class="QGroupBox" name="logBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>761</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Log</string>
   </property>
   <widget class="QListWidget" name="log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>741</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="statusBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>67</height>
    </rect>
   </property>
   <property name="title">
    <string>Status</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Chat</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Stream</string>
    </property>
   </widget>
   <widget class="QLabel" name="chatConnectedLabel">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 212, 212);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Disconnected</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="streamOnlineLabel">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 212, 212);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Offline</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="updateStreamStateButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>38</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="app.qrc">
      <normaloff>:/media/media/reload.png</normaloff>:/media/media/reload.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="streamInfoBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>181</width>
     <height>67</height>
    </rect>
   </property>
   <property name="title">
    <string>Stream Info</string>
   </property>
   <widget class="QLabel" name="fpsLabel">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="viewersLabel">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>40</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Viewers:</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>761</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Auth</string>
    </attribute>
    <widget class="QGroupBox" name="loginBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>721</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>Log in panel</string>
     </property>
     <widget class="QLineEdit" name="usernameEdit">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Channel to connect</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="channelEdit">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string> Username</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>90</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="getAuthButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Auth Token</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Put token here:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="authTokenEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="forceReconnectBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>110</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Force Reconnect</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="app.qrc">
        <normaloff>:/media/media/save.png</normaloff>:/media/media/save.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>31</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Bot Settings</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>15</y>
       <width>471</width>
       <height>239</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Predefined commands</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="commandsTable">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>5</y>
         <width>311</width>
         <height>201</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <column>
        <property name="text">
         <string>Command</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>On/Off</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Min Delay (msec)</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Custom commands</string>
      </attribute>
      <widget class="QTableWidget" name="customCommandsTable">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>5</y>
         <width>421</width>
         <height>201</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <column>
        <property name="text">
         <string>Command</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>On/Off</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Min Delay (msec)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Message</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="announcementBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Announcement</string>
     </property>
     <widget class="QTextEdit" name="announcementTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="repeatAnnouncementCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>82</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Repeat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendStopAnnouncementButton">
      <property name="geometry">
       <rect>
        <x>137</x>
        <y>79</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="announcementRepeatTime">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Test Twitch API</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Last 10 followers</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lastFollowers">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Subscriber Actions</string>
    </attribute>
    <widget class="QGroupBox" name="greetingNewSubBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QTextEdit" name="greetingMsgTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;@sub, thanks for (re)subscribing, mate!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="greetNewSubCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Greet new subscriber</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="chatBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>351</width>
     <height>103</height>
    </rect>
   </property>
   <property name="title">
    <string>Chat</string>
   </property>
   <widget class="QListWidget" name="chat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>73</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget::item { border-bottom: 1px solid black; }</string>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="aboutLabel">
   <property name="geometry">
    <rect>
     <x>766</x>
     <y>1</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>https://github.com/degot88/DigiBot</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="app.qrc">:/media/media/question_mark.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>connectButton</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>forceReconnectBox</tabstop>
  <tabstop>updateStreamStateButton</tabstop>
  <tabstop>authTokenEdit</tabstop>
  <tabstop>channelEdit</tabstop>
  <tabstop>log</tabstop>
  <tabstop>getAuthButton</tabstop>
  <tabstop>usernameEdit</tabstop>
  <tabstop>chat</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>customCommandsTable</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lastFollowers</tabstop>
  <tabstop>tabWidget_2</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>commandsTable</tabstop>
  <tabstop>announcementTextEdit</tabstop>
  <tabstop>repeatAnnouncementCheckBox</tabstop>
  <tabstop>announcementRepeatTime</tabstop>
  <tabstop>sendStopAnnouncementButton</tabstop>
  <tabstop>greetNewSubCheckBox</tabstop>
  <tabstop>greetingMsgTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
